modules/1_bastion/variables.tf

```diff
@@ -65,23 +65,23 @@ variable "ansible_repo_name" {
 
 variable "rhel_subscription_username" {
   default   = ""
-  sensitive = true
+  #sensitive = true
```

Comment:
```suggestion
```

Reply:
Let's remove the comments

Reply:
Removed the commented line for sensitive variable